### ISSUE TITLE
Why does this exist?

### ISSUE THREAD

**Issue by NullVoxPopuli** [pete-the-pete/simple-ember-language-server#10]
We now have 4 language servers:
 - https://github.com/ember-tooling/ember-language-server (not maintained)
 - https://github.com/lifeart/ember-language-server (actually maintained)
 - https://github.com/suchitadoshi1987/ember-language-server (fork of the maintained)
 - and now this one.
 
 Some questions:
  - why not contribute the the most up to date language server? -- lifeart/ember-language-server
  - what's lifeart/ember-language-server missing that this language server provides?
  - if all 4 of these are on the VSCode marketplace, _I feel_ like it makes the community look a bit disorganized, and like we can't agree on anything, and I just want to know why all the forking and lack of collaboration :(

**Comment by pete-the-pete** [comment]
I agree. I'm not certain I'll ever publish this to the Marketplace. I'd much rather fold it in to an existing option, or replace https://github.com/suchitadoshi1987/ember-language-server, if it ever got that far.

I wanted something I could use on my larger project, https://github.com/lifeart/ember-language-server and https://github.com/suchitadoshi1987/ember-language-server are too slow, offer features provided by other means (eslint, jsconig, typescript) and support old ember patterns I don't have in my app.
I started by contributing on https://github.com/suchitadoshi1987/ember-language-server, but realized I could get something useful by starting fresh, and potentially folding it back in later.

**Comment by lifeart** [comment]
Hi @pete-the-pete!

Could you add more details about slowness?

https://github.com/lifeart/ember-language-server has list of settings to do less work for large projects:

- [els.local.collectTemplateTokens](https://github.com/lifeart/vscode-ember/blob/master/package.json#L65 ) - you could disable template indexing for related files
- [els.local.ignoredProjects](https://github.com/lifeart/vscode-ember/blob/master/package.json#L75) - you could exclude projects/addons from analyzis
- [els.local.useBuiltinLinting](https://github.com/lifeart/ember-language-server/blob/component-context-info-origin/src/server.ts#L131) - to disable template linting

Also, uELS has addons api, and you literally coud create addon and replace all builtin functionality (it's possible)

**Comment by suchitadoshi1987** [comment]
@pete-the-pete, I agree with @NullVoxPopuli - we may want to explore the option to merge your changes in one of the actively maintained ELS. 
The latest version of https://github.com/suchitadoshi1987/ember-language-server should speed things up specially since the way our project is now setup. Lets sync on that and determine the next steps.
